haroonmaqsoodch/ShopGo
# [SG-11]: Create onboarding screens

## jira
[SG-11](https://shopgoswift.atlassian.net/browse/SG-11)

Description:
- Create onboarding screens with swipe-able screens
- The onboarding screen should only be shown once 
- Opening the app next time should take you directly to the login screen

how to test ;
- run the app normally the first time you open it up you should see the onboarding screens, close the app and relaunch it which should take you directly to the login screen after the start-up animation

screenshots:

https://github.com/haroonmaqsoodch/ShopGo/assets/63094797/499add7a-0061-4eea-bcd0-40609061f52d
